.github/workflows/golangci-lint.yml

```diff
@@ -21,4 +21,4 @@ jobs:
         uses: golangci/golangci-lint-action@v2
```

Comment:
I found this repo: https://github.com/golangci/golangci-lint-action

Looks like they have a `@v3` now... They have a thing in their README saying how to configure it: https://github.com/golangci/golangci-lint-action#compatibility

(In case you couldn't tell, I have never worked with this Github CI configuration either, so I'm suggesting things ad-hoc)

Reply:
Thanks for the help. This is new to me as well so let me read up on it a little. Btw, this test isn't marked as required. Does that mean it would be ok to merge this PR and fix it in a follow up? Ideally, I would like to fix it before merge but wanted to know what my options are in case debugging this takes too long.

Reply:
Interestingly its running a go version of 1.18 even though go mod specifies it as 1.17 
```
 GOVERSION="go1.18.2"
```

I think I need to understand the setup better.